adi/sync_start.py

```diff
@@ -55,6 +55,50 @@ def tx_sync_start_available(self):
         except:  # noqa: E722
             return "arm"
 
+    @property
+    def tx_b_sync_start(self):
+        """ tx_b_sync_start: Issue a synchronisation request
+
+        Possible values are:
+            - **arm**: Writing this key will arm the trigger mechanism sensitive to an
+              external sync signal. Once the external sync signal goes high
+              it synchronizes channels within a DAC, and across multiple
+              instances. This bit has an effect only the EXT_SYNC
+              synthesis parameter is set.
+            - **disarm**: Writing this key will disarm the trigger mechanism
+              sensitive to an external sync signal. This bit has an
+              effect only the EXT_SYNC synthesis parameter is set.
+            - **trigger_manual**: Writing this key will issue an external sync event
+              if it is hooked up inside the fabric. This key has an effect
+              only the EXT_SYNC synthesis parameter is set.
+              This key self clears.
+        """
+        try:
+            return self._get_iio_dev_attr_str("sync_start_enable", _ctrl=self._txdac2)
+        except:  # noqa: E722
+            if self._txdac2.reg_read(0x80000068) & 0x1 == 0x1:
+                return "arm"
+            else:
+                return "disarm"
```

Comment:
Something like this is better to handle the backwards compatbility instead of try catch
```
    if "sync_start_enable" in self._ctrl.attrs:
            return self._get_iio_dev_attr_str("sync_start_enable", _ctrl=self._txdac2)
    else:
            if self._txdac2.reg_read(0x80000068) & 0x1 == 0x1:
                return "arm"
            else:
                return "disarm"
```